ui/src/embeddable/filter/countrySelector/CountrySelector.jsx

```diff
@@ -65,7 +65,7 @@ const CountrySelector = ({ countries, filters, onApply, selectedCountryFirst, ad
           <Accordion.Title
             active={activeIndex === 0}
             content='Select a country'
-            icon="angle right"
+            icon="caret down"
```

Comment:
changing the icon changes the one for the data summary. also it does not change when the dropdown is expanded

Reply:
@jdeanquin-dg Restored previous arrow icon.